FirebaseAuth/Sources/Swift/Utilities/AuthErrors.swift

```diff
@@ -46,7 +46,9 @@ import Foundation
    associated with this key is an NSString of the email address of the account that already
    exists.
    */
-  @objc(FIRAuthErrorUserInfoEmailKey) static var AuthErrorUserInfoEmailKey: String =
+  @objc(
```

Comment:
nit: reformat with `@objc(FIRAuthErrorUserInfoEmailKey)` on a single predecessor line.

Reply:
Done here and everywhere.